include/zephyr/bluetooth/mesh/health_srv.h

```diff
@@ -171,18 +166,18 @@ struct bt_mesh_health_srv {
  *
  *  @param srv Pointer to a unique struct bt_mesh_health_srv.
  *  @param pub Pointer to a unique struct bt_mesh_model_pub.
+ *  @param ... Optional Health Server metadata, if compiled with Large
+ *             Composition Data Server support, otherwise this parameter is
+ *             ignored.
```

Comment:
```suggestion
 *  @param ... Optional Health Server metadata if the model is compiled with Large
 *             Composition Data Server support, otherwise this parameter is
 *             ignored.
```